### PR TITLE
Revised .gitignore

### PR DESCRIPTION
**Pull request by MajorMurphy**
Ignoring build files from Xcode and Visual Studio

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,8 +41,9 @@ Testing/IccToXml
 # Xcode deps
 Build/XCode/libtiff/*.h*
 Build/XCode/libxml/*.h*
+
 # Xcode build files
-**/build
+**/*.noindex
```
Comment (beku) [inline .gitignore:46]: Please keep the build path ignore part. It is used for CMake builds.

Comment (MajorMurphy) [reply .gitignore:46]: The project root has a folder named [Build](https://github.com/InternationalColorConsortium/RefIccMAX/tree/master/Build), which then is undesirably ignored.

Comment (MajorMurphy) [reply .gitignore:46]: Resolved in 716d3e5 by ignoring all build folders except that one.

**Comment by beku** [comment]
Thanks!

**Pull request merged**